packages/oui-action-menu/README.md

```diff
@@ -0,0 +1,75 @@
+# Action menu
+
+<component-status cx-design="complete" ux="prototype"></component-status>
+
+## Usage
+
+### Normal
+
+```html:preview
+<oui-action-menu
+  text="Actions"
+  aria-label="Server: actions"
+  align="start">
+  <oui-action-menu-item
+    text="Action 1"
+    aria-label="Server: action 1"
+    on-click="$ctrl.showCurrentValueInPopup('Action 1')"></oui-action-menu-item>
+  <oui-action-menu-item
+    text="Action 2"
+    aria-label="Server: action 2"
+    on-click="$ctrl.showCurrentValueInPopup('Action 2')"></oui-action-menu-item>
+  <oui-action-menu-divider></oui-action-menu-divider>
+  <oui-action-menu-item
+    text="Lien externe"
+    href="#"
+    external></oui-action-menu-item>
+</oui-action-menu>
+```
+
+### Compact
+
+```html:preview
+<div class="oui-action-menu-container">
+  <oui-action-menu
+    text="Actions"
```

Comment:
What do you do with text in compact mode ?

Reply:
I remove it but now, if `aria-label` is not filled, it will be replaced by `text`.